search/test.py

```diff
@@ -0,0 +1,56 @@
+import json
+from unittest import TestCase
+
+from search.views import *
+from user.models import User
+from core import app
+
+
+class SearchTest(TestCase):
+    def setUp(self):
+        self.client = app.test_client()
+
+    def tearDown(self):
+        User.objects.delete()
+        Article.objects.delete()
+
+    def test_search_retrieve_several_user(self):
+        User(email='john@test.com',
+             username='john').set_password('search').save()
+        User(email='johndoe@test.com',
+             username='johndoe').set_password('search').save()
+        response = self.client.get('/search/john')
```

Comment:
We probably want to call the _GET_ with a search in a query parameters: `/search/?q=john`